test/unit_tests/braket/default_simulator/cirq/test_openqasm_conversion.py

```diff
@@ -0,0 +1,279 @@
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+            ],
+        ),
+        (
+            """
+        OPENQASM 3.0;
+        bit[3] b;
+        qubit[3] q;
+        cswap q[0], q[1], q[2];
+        b[0] = measure q[0];
+        b[1] = measure q[1];
+        b[2] = measure q[2];
+        """,
+            [
+                1.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+            ],
+        ),
+        (
+            """
+                        OPENQASM 3.0;
+                        bit[5] b;
+                        qubit[5] q;
+                        rx(0.15) q[0];
+                        ry(0.2) q[1];
+                        rz(0.25) q[2];
+                        h q[3];
+                        cnot q[0], q[2];
+                        cnot q[1], q[3];
+                        x q[1];
+                        x q[3];
+                        y q[1];
+                        y q[3];
+                        z q[1];
+                        z q[2];
+                        s q[2];
+                        t q[1];
+                        cphaseshift(0.15) q[2], q[4];
+                        swap q[3], q[4];
+                        ccnot q[2], q[3], q[4];
+                        cswap q[1], q[3], q[4];
+                        b[0] = measure q[0];
+                        b[1] = measure q[1];
+                        b[2] = measure q[2];
+                        b[3] = measure q[3];
+                        b[4] = measure q[4];
+                        """,
+            [
+                -0.69612223 + 0.08747133j,
+                0.69612223 - 0.08747133j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                -0.05559387 - 0.04318215j,
+                0.0 + 0.0j,
+                0.05559387 + 0.04318215j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.05230728 - 0.00657268j,
+                -0.05230728 + 0.00657268j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.0 + 0.0j,
+                0.00417738 + 0.00324475j,
+                0.0 + 0.0j,
+                -0.00417738 - 0.00324475j,
+                0.0 + 0.0j,
+            ],
+        ),
+    ),
+)
+def test_openqasm_conversion_to_cirq(openqasm, expected_result):
```

Comment:
How come we're passing in expected result as an argument? It seems it should be pretty easy to generate from the OpenQASM in the test

Reply:
I did not know we can generate directly final state vector from OpenQASM.
Could you send me a reference?

I generated all the expected result using the state vector simulator

Reply:
Yes exactly, I'm saying you can do the same thing (as I linked in another comment) in the test

Reply:
I thought that would create a dependency. But it makes sense for future testcases